models/auth/token.go

```diff
@@ -21,6 +20,34 @@ import (
 	lru "github.com/hashicorp/golang-lru"
 )
 
+// ErrAccessTokenNotExist represents a "AccessTokenNotExist" kind of error.
+type ErrAccessTokenNotExist struct {
```

Comment:
I have a slight suspicion that I'll encounter this code snippet again when reviewing #20891…